package.json

```diff
@@ -4,10 +4,10 @@
   "description": "The frontend part of Redash.",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server",
+    "start": "node --max-old-space-size=4096 node_modules/.bin/webpack-dev-server",
```

Comment:
😮 build was failing without this? somewhat disturbing.

Reply:
I backed this out and CircleCI started failing with heap space exhausted error again, so I think so. Not sure if there's something we can do to get webpack to run leaner.